egg-herbie/Cargo.toml

```diff
@@ -28,4 +28,3 @@ opt-level = 1
 [profile.release]
 debug = true
 lto = "fat"
-codegen-units = 1
```

Comment:
Any idea what this does? Do LTO across egg? Why change it?

Reply:
Oops, will remove.